test/iaas.GcpClient.spec.js

```diff
@@ -0,0 +1,88 @@
+'use strict';
+
+const lib = require('../lib');
+const GcpClient = lib.iaas.GcpClient;
+
+describe('iaas', function () {
+  describe('GcpClient', function () {
+
+    const config = {
```

Comment:
All methods can be UTed via the usage of at-least stubs, if not via nocks (external http)